bundles/org.openhab.binding.solaredge/src/main/resources/OH-INF/i18n/solaredge.properties

```
@@ -129,3 +129,13 @@ channel-type.solaredge.type-energy.label = Energy
 channel-type.solaredge.type-percent.label = Percent
 channel-type.solaredge.type-power.label = Power
 channel-type.solaredge.type-status.label = Status Text
+
+# status translations
+
+status.invalid.solarId = solarId is either invalid or belongs to another account.
+status.invalid.token = the token seems to be invalid.
+status.unkown.error = unkown error
+status.invalid.token.length = you will have to use a 'token' and not an 'api key' when using private API.
+status.invalid.api.key.length = you will have to use an 'api key' and not a 'token' when using public API.
+status.request.limit.exceeded = daily request limit (300) exceeded.
```

```suggestion
status.request.limit.exceeded = Daily request limit (300) exceeded.
```

consider a placeholder for the value 300, so that retranslation will not be needed when/if it changes.

---

Can you give me a hint how to do this? The documentation does not contain any information about this special topic: https://www.openhab.org/docs/developer/utils/i18n.html

---

https://github.com/openhab/openhab-addons/blob/f2e7597db69598b40b7f9bae24c7ac5dedabb345/bundles/org.openhab.binding.danfossairunit/src/main/java/org/openhab/binding/danfossairunit/internal/handler/DanfossAirUnitHandler.java#L108

---

ok I thought there was some special fancy way to define placeholders in the i18n texts.

---

The corresponding text:
https://github.com/openhab/openhab-addons/blob/f2e7597db69598b40b7f9bae24c7ac5dedabb345/bundles/org.openhab.binding.danfossairunit/src/main/resources/OH-INF/i18n/danfossairunit.properties#L88

---

Well the java code looks like it is just appended as text...